lab_1/main.py

```diff
@@ -25,7 +31,12 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
+    tokens_clean = []
+    if isinstance(tokens, list):
```

Comment:
Usually, there are checks for arguments and body function - separated and on the same level.
Try to get rid of nesting.